### DISCUSSION TITLE
Ingress TCP ports

### DISCUSSION THREAD

**Discussion by lordigon** [arkmq-org/arkmq-org-broker-operator#594] [Q&A] [answered]
Is it possible to use a different TCP ports exposed by the ingress for TLS traffic?
By default when you expose an acceptor with SSL enabled the TCP port exposed by the ingress is 443 but I would like to have the chance to change it.

**Comment by gaohoward** [comment, marked as answer]
I don't think it's possible right now. If you look at the ingress doc https://kubernetes.io/docs/concepts/services-networking/ingress/ the ingress TLS only support single TLS port which is 443
